test/integration/page-config/test/index.test.js

```diff
@@ -13,6 +13,16 @@ describe('Page Config', () => {
   it('builds without error when export const config is used outside page', async () => {
     const { stderr } = await nextBuild(appDir, undefined, { stderr: true })
     expect(stderr).not.toMatch(/Failed to compile\./)
+
+    const clientFileOutput = await fs.readFile(
+      join(
+        appDir,
+        '.next/static/test-page-config',
+        'pages/check-client-remove.js'
+      ),
+      'utf8'
+    )
+    expect(clientFileOutput).not.toMatch('THIS_VALUE_SHOULD_BE_GONE')
```

Comment:
This seems ok for now, although we might want to add showing a warning if they add an unexpected page config value in the future 🤔

Reply:
We can deal with that when we get there!